content/docs/instrumenting/standard_exports.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Standard Exports
+sort_rank: 7
+---
+
+# Standard Exports
+
+This document covers what exports and collectors Prometheus client libraries
+should offer, with the aim of consistency across libraries, making the easy use
+cases easy and making it easier to provide tooling and consoles for common use
+cases.
+
+In addition, client libraries are ENCOURAGED to also offer whatever makes sense
+in terms of metrics for their language’s runtime (e.g. Garbage collection
+stats).
+
+These exports should have the prefix `process_`. If a language or runtime
```

Comment:
We need more clarity here. The unified process metrics are prefixed `process_` but the language specific metrics are prefixed `<language>_` or similar (e.g. `go_goroutines`, but note that JVM metrics are prefixed `hotspot_`).